python/pylibcudf/pylibcudf/io/avro.pyx

```diff
@@ -141,7 +143,9 @@ cpdef TableWithMetadata read_avro(
     options: AvroReaderOptions
         Settings for controlling reading behavior
     """
+    if stream is None:
+        stream = Stream()
```

Comment:
We need to streamline this. Maybe we can use a default besides `None`, or perhaps we need a `cdef stream_or_default(stream)` function.